package Error

func Throw(v uint) {
	panic(v)
}
